src/modules/item/actions.ts

```diff
@@ -64,6 +64,32 @@ export type SaveItemRequestAction = ReturnType<typeof saveItemRequest>
 export type SaveItemSuccessAction = ReturnType<typeof saveItemSuccess>
 export type SaveItemFailureAction = ReturnType<typeof saveItemFailure>
 
+// Save multiple items
+
+export const SAVE_MULTIPLE_ITEMS_REQUEST = '[Request] Save Multiple Items'
+export const SAVE_MULTIPLE_ITEMS_SUCCESS = '[Success] Save Multiple Items'
+export const SAVE_MULTIPLE_ITEMS_FAILURE = '[Failure] Save Multiple Items'
+export const SAVE_MULTIPLE_ITEMS_CANCELLED = '[Cancelled] Save Multiple Items'
+export const SAVE_MULTIPLE_ITEMS_CANCEL = '[Cancel] Save Multiple Items'
+export const SAVE_MULTIPLE_ITEMS_CLEAR_STATE = '[Clear State] Save Multiple Items'
```

Comment:
I see the value of having these two actions start with the same prefix as the other parts of the flow. BUT I think it's a bit misleading. They're not part of the async flow of saving items, so maybe starting with the verb is more obvious?

```ts
export const CANCEL_SAVE_MULTIPLE_ITEMS = '[Cancel] Save Multiple Items'
export const CLEAR_SAVE_MULTIPLE_ITEMS = '[Clear] Save Multiple Items'
```

I've purposefully omitted `state` there. I think that, _maybe_, it's not necessary

The types and actions are already following this pattern btw